pkg/api/aim/dashboards.go

```diff
@@ -22,10 +23,18 @@ func GetDashboards(c *fiber.Ctx) error {
 
 	var dashboards []database.Dashboard
 	if err := database.DB.
-		Preload("App").
-		Joins("LEFT JOIN apps ON apps.id = dashboards.app_id").
+		InnerJoins(
+			"App",
+			database.DB.Select(
+				"ID", "Type", "IsArchived",
```

Comment:
We don't need to select `IsArchived` here

Reply:
yep, it was just before. I can remove it.

Reply:
done.